src/views/AlbumContent.vue

```diff
@@ -309,24 +309,17 @@ export default {
 				return []
 			}
 
-			const semaphoreSymbol = await this.semaphore.acquire(() => 0, 'fetchFiles')
-			const fetchSemaphoreSymbol = await this.fetchSemaphore.acquire()
```

Comment:
Semaphores are used to prioritize XHR requests compared to photos requests. Removing it will disable this functionality

Reply:
The PQueue has a priority argument, which could be used for this. Then we wouldn't have prioritization littered over the code base. What do you think about that?

Reply:
Priority of potos loading might change as the user scrolls.

If we want to prioritize xhr over photos, and have dynamic priority for photos, then semaphore is a good fit

Reply:
I see. I've removed my changes to the semaphore code